### PR TITLE
Tickets/dm 38930

### PR DESCRIPTION
**Pull request by edennihy**

### CODE REVIEW

File: doc/version_history.rst [hunk 1]
```diff
@@ -6,10 +6,17 @@
 Version History
 ===============
 
+v1.21.0
```
Comment (tribeiro) [inline doc/version_history.rst:9]: We have not released 1.20.0 yet, so you can make these new scripts part of that release instead.

File: python/lsst/ts/standardscripts/base_offset_tcs.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        self.reset_offsets = getattr(config, "reset_offsets", None)
+
+        self.relative = config.relative
+        self.absorb = config.absorb
+
+    def set_metadata(self, metadata):
+        metadata.duration = 10
+
+    async def assert_feasibility(self) -> None:
+        """Verify that the telescope and is in a feasible state to
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/base_offset_tcs.py:158]: remove the "and" after "telescope"?


> Verify that the telescope ~and~ is in a feasible...

File: python/lsst/ts/standardscripts/base_offset_tcs.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        execute the script.
+        """
+
+        await self.tcs.assert_all_enabled()
+
+    async def run(self):
+        await self.assert_feasibility()
+
+        if self.offset_azel:
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/base_offset_tcs.py:167]: I think it is better if you check that this is not None. Doing it this way may give the impression that `self.offset_azel` is a boolean. Also, I think it is not safe to assume that `None==False`.

Consider changing these to:

```
if self.offset_azel is not None:
    ...
```

**Review by tribeiro — COMMENTED** [review]
Here some initial comments. I want to double check this configuration before approving the PR.

File: tests/test_maintel_offset_mtcs.py [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+    async def test_offset_rot(self):
+        async with self.make_script():
+            config = dict(offset_rot=dict(rot=10))
+
+            await self.configure_script(**config)
+
+            self.script.mtcs.offset_rot = unittest.mock.AsyncMock()
+
+            await self.script.run()
+
```
Comment (tribeiro) [inline tests/test_maintel_offset_mtcs.py:165]: Make sure you assert that `self.script.mtcs.offset_rot` was awaited with the expected value.

**Review by tribeiro — APPROVED** [review]
Alright! I verified the configuration schema is actually doing what we need it to do, so all good!

One additional comments for you to consider before merging.

File: doc/version_history.rst (hunk 1 above)
Comment (edennihy) [reply doc/version_history.rst:9]: Updated to v1.20.1

File: tests/test_maintel_offset_mtcs.py (hunk 4 above)
Comment (edennihy) [reply tests/test_maintel_offset_mtcs.py:165]: I think that's being done in L166 below, right?

**Pull request merged**